CHANGELOG.md

```diff
@@ -1,216 +1,51 @@
-# Release 12.0.0
+# Release 12.1.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio and Venia 12.0.0._
-_For older release notes, see [PWA Studio releases][]._
+_This changelog only contains release notes for PWA Studio and Venia 12.1.0_
+_For older release notes, see_ [PWA Studio releases][].
+
+| Type  | Status       | Assignee                 | Description                                                                           | Jira Issue   | GitHub PR       | Release    |
+| :---- | :----------- | :----------------------- | :------------------------------------------------------------------------------------ | :----------- | :-------------- | :--------- |
+| Task  | Done         |                          | Publish new Magento PWA meta package https://github.com/magento-commerce/magento2-pwa | [PWA-2329][] | No PR           | PWA-12.1.0 |
+| Bug   | Deployment Q | Tommy Wiebell            | Cypress snapshots outdated with newsletter in footer                                  | [PWA-2287][] | [3506][]        | PWA-12.1.0 |
+| Bug   | Deployment Q | Devagouda Patil          | Venia Mega nav is broken in scaffolded app on develop branch.                         | [PWA-2286][] | [3513][]        | PWA-12.1.0 |
+| Task  | Deployment Q | Devagouda Patil          | Repo Metadata Service Onboarding: UPWARD-PHP                                          | [PWA-2251][] | [5][]           | PWA-12.1.0 |
+| Bug   | Deployment Q | Justin Conabree          | Cart remains active in browser memory on PWA site even after checkout                 | [PWA-2190][] | [3495][]        | PWA-12.1.0 |
+| Story | Deployment Q | Tommy Wiebell            | Parallelize Cypress tests on the CI                                                   | [PWA-2154][] | [3460][]        | PWA-12.1.0 |
+| Story | Deployment Q | James Calcaben           | Venia Sample Data metapackage deployVeniaSampleData.sh script update                  | [PWA-2151][] | [3473][]        | PWA-12.1.0 |
+| Task  | Deployment Q | Tommy Wiebell            | Add installation instructions to Venia Sample Data repo                               | [PWA-2141][] | [8][]           | PWA-12.1.0 |
+| Task  | Deployment Q | Bruce Denham             | Add local and Cloud installation instructions to meta packages                        | [PWA-2140][] | [4][], [1][]    | PWA-12.1.0 |
+| Bug   | Deployment Q |                          | Cart page out of stock crash                                                          | [PWA-2133][] | [3447][], [6][] | PWA-12.1.0 |
```

Comment:
@bdenham @supernova-at Is it good to mention magento/pwa meta package dependency on this?

Reply:
@dpatil-magento Good question. One I was going to ask myself. I'll remove for now.

Reply:
It would be great to mention it, yeah. Especially if we could include installation instructions (or a link to them).

"`12.1.0` introduces the metapackage, and future PWA Studio features will depend on this metapackage being present" or something similar